administration/src/bp-modules/cards/ImportCardsFromFreinetController.ts

```diff
@@ -0,0 +1,66 @@
+import { BAVARIA_CARD_TYPE_GOLD, BAVARIA_CARD_TYPE_STANDARD } from '../../cards/extensions/BavariaCardTypeExtension'
+import { ProjectConfig } from '../../project-configs/getProjectConfig'
+
+const FREINET_EXPIRATION_DATE_COLUMN_HEADER = 'eak_datum'
+const FREINET_FIRSTNAME_COLUMN_HEADER = 'vorname'
+const FREINET_LASTNAME_COLUMN_HEADER = 'nachname'
+const FREINET_CARDTYPE_COLUMN_HEADER = 'inhaber_ehrenamtskarte'
+
+const mergeFirstAndLastnameIntoNewColumn = (line: string[], csvHeader: string[], nameColumnName: string) => {
```

Comment:
:upside_down_face: Is all of this done in place as no copies or anything are returned? Wouldn't it be a lot safer to work with copies? Looks kindof error prone to me.
E.g. if the header is adjusted in an early iteration, doesn't that have implications for later iterations?

Reply:
Yes it does impact later iterations, but this is handled and works as expected. You are right it may be not the best implementation, but we are not planning to have this feature for a ever or a long time, it is just a workaround, we will remove this as soon as Freinet and entitlemtcard are connected.